src/main/webapp/app/tasks/task-search/task-search.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TaskSearchComponent } from './task-search.component';
+
+describe('TaskSearchComponent', () => {
+  let component: TaskSearchComponent;
+  let fixture: ComponentFixture<TaskSearchComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ TaskSearchComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(TaskSearchComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
actual test cases would have been great

Reply:
I've written some test cases for the whole component, I wish could cover all possible cases and component functionality.